web/themes/custom/cloudy/templates/content/events/node--event.html.twig

```diff
@@ -21,23 +21,35 @@
           <div class="event_time d-flex">
               {% if content.field_all_day|render|striptags|trim is not empty %}
                 {{ content.field_all_day }}
-              {% elseif content.field_end_time|render|striptags|trim is not empty %}
-                <time datetime="{{ content.field_start_time.0 }}">{{ content.field_start_time }}</time>
-                <span aria-label="{{ 'to'|t }}"> &ndash; </span>
-                <time datetime="{{ content.field_end_time.0 }}">{{ content.field_end_time.0 }}</time>
+              {% elseif content.field_start_time.0|render|striptags|trim is not empty and content.field_end_time.0|render|striptags|trim is not empty %}
+                {{ content.field_start_time }}
+                <span class="mx-2" aria-label="{{ 'to'|t }}"> &ndash; </span>
+                {{ content.field_end_time }}
+              {% elseif content.field_start_time.0|render|striptags|trim is empty and content.field_end_time.0|render|striptags|trim is not empty %}
+                <span class="mr-1">Ends </span>
```

Comment:
Which content type does "Ends " apply to again?

Reply:
Ends is for when bids and proposals only put in an end time and no start time. It represents the closing of a big or proposal period.

Reply:
Odd I can't comment on your comment about "more about this location". 🤔 

I guess I'll just add it here. That secondary more about this location is on the location (building with address) rather than on the service location (place with operational hours). 

That could definitely get a refactor but it is out of the scope for what I was trying to do here. I'll add that to the follow up where we are looking at translation on that field since we'll touch all those more links that that time.